tests/Fixtures/app/config/config_common.yml

```diff
@@ -86,10 +92,6 @@ api_platform:
     mercure:
         include_type: true
 
-parameters:
-    container.autowiring.strict_mode: true
-    .container.dumper.inline_class_loader: true
```

Comment:
these should be changed in the 3.1 branch no?

Reply:
It is just moved (to line 1), not removed